packages/builder/src/components/automation/SetupPanel/AutomationBlockSetup.svelte

```diff
@@ -317,210 +324,233 @@
 
 <div class="fields">
   {#each deprecatedSchemaProperties as [key, value]}
-    <div class="block-field">
```

Comment:
Do you know what formatting change occurred here for this file to be so different?

Reply:
I think it's because of the new line `{#if canShowField(key, value)}` which wraps the entire field block, so all the child divs etc got shifted inwards.

Reply:
Ah that makes sense, good old git freaking out lol